src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
```

Comment:
Yes, this script is linux only. The OSX version of the script is on the way. It is tracked via #6161